doc/faq.rst

```diff
@@ -176,9 +176,7 @@ Using Spyder
       :align: left
       :start: 306
 
-  |
-
-   If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**). For Spyder to recognize it, the installation should be done in one of the following default paths:
+   | If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**). For Spyder to recognize it, the installation should be done in one of the following default paths:
```

Comment:
```suggestion
   |

   If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**).
   For Spyder to recognize it, the installation should be done in one of the following default paths:
```

One Line Per Sentance (this does inject some extra vertical space, but that will be fixed by removing the `|` and instead making the CSS change).

Reply:
Hi @CAM-Gerlach glad to see you back! Regarding this suggestion, when I apply this change in my local I see a bigger space that the one you can see with the initial fix of putting the `|` in the correct place (as you describe in your comment):

Initial fix:
![image](https://user-images.githubusercontent.com/16781833/134140832-db449b40-5699-4793-80f9-bddf30f6116c.png)


With the suggestion:

![image](https://user-images.githubusercontent.com/16781833/134140371-345d89ae-7250-4224-b2d1-5de0d6bfa25a.png)

However I'm not totally sure but is that actually what you want? (to leave that space) If you put the `|` in the correct place you can have one line per sentence and the correct spacing between the video and the text.

Reply:
Or maybe I just should do the CSS addition here and remove the `|`?

Reply:
Good to see you again too!

> Or maybe I just should do the CSS addition here and remove the `|`?

Sorry for the confusion. Yeah, that was what I intended; this suggestion was an alternative if you wanted to have me do the CSS change in another PR. Making the first sentance a block with the `|` would be okay too, it just adds an extra div around that sentence that adds margin both above and below that doesn't exactly match the following sentence, but either way it doesn't matter since it would be shortly fixed anyway by my followup PR. So up to you. Here's the alternative for reference (not directly tested).

```suggestion
   | If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**).
   For Spyder to recognize it, the installation should be done in one of the following default paths:
```